izpack-panel/src/main/java/com/izforge/izpack/panels/userinput/gui/text/GUITextArea.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * IzPack - Copyright 2001-2013 Julien Ponge, All Rights Reserved.
+ *
+ * http://izpack.org/
+ * http://izpack.codehaus.org/
+ *
+ * Copyright 2020 Patrick Reinhart
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.izforge.izpack.panels.userinput.gui.text;
+
+import static javax.swing.ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER;
+import static javax.swing.ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED;
+
+import com.izforge.izpack.api.handler.Prompt;
+import com.izforge.izpack.panels.userinput.field.Field;
+import com.izforge.izpack.panels.userinput.field.ValidationStatus;
+import com.izforge.izpack.panels.userinput.field.text.TextArea;
+import com.izforge.izpack.panels.userinput.field.text.TextField;
+import com.izforge.izpack.panels.userinput.gui.GUIField;
+
+import javax.swing.*;
```

Comment:
I would narrow this down. And awt.* too.

Reply:
Fixed those...